app/controllers/LanguageSwitchController.scala

```diff
@@ -30,7 +30,7 @@ class LanguageSwitchController @Inject()(
                                           val controllerComponents: MessagesControllerComponents
                                         ) extends FrontendBaseController with I18nSupport {
 
-  private def fallbackURL: String = routes.IndexController.onPageLoad().url
+  private def fallbackURL: String = routes.WhatYouNeedController.onPageLoad().url
 
   private def languageMap: Map[String, Lang] = appConfig.languageMap
 
```

Comment:
I think we may be able to remove this controller also given the service is not multi lingual.